### PR TITLE
Quote for both run and debug so it.each works

### PR DESCRIPTION
**Pull request by tlbdk**
``` typescript
it.each([1,2,3,4])('should parse and generate %i and generate it again', id => {
...
})
```

Will generate the following command run:

```
node 'node_modules/.bin/jest' '/Users/tlb/git/connectedcars/ingester-tl-v2/src/lib/traffilog-parser/tools/monitor-csv-parser.test.ts' -t 'parser should parse and generate (.*?) and generate it again'
```

and the following command with debug:
```
... node node_modules/.bin/jest --testTimeout=100000000 --detectOpenHandles /Users/tlb/git/connectedcars/ingester-tl-v2/src/lib/traffilog-parser/tools/monitor-csv-parser.test.ts -t parser\ should\ parse\ and\ generate\ \(.\*?\)\ and\ generate\ it\ again --runInBand 
```

Resulting in:
```
zsh: no matches found: parser should parse and generate setting (.*?) and generate it again
```

### CODE REVIEW

**Comment by firsttris** [comment]
thanks for your contribution

**Comment by fenixil** [comment]
@firsttris hello, I've stumbled upon this issue too and glad to see that fix exists. Kindly advice if you plan to accept this PR.

**Review by domsleee — APPROVED** [review]

File: src/jestRunner.ts [hunk 1]
```diff
@@ -177,7 +177,7 @@ export class JestRunner {
       config.program = `.yarn/releases/${this.config.getYarnPnpCommand}`;
     }
 
-    const standardArgs = this.buildJestArgs(filePath, currentTestName, false);
+    const standardArgs = this.buildJestArgs(filePath, currentTestName, true);
```
Comment (domsleee) [inline src/jestRunner.ts:180]: actually oops, this breaks the given example on windows + powershell and windows + cmd
Reason is this line:
https://github.com/firsttris/vscode-jest-runner/blob/e3f7c76334c248ff426fd078a7634213b4106174/src/util.ts#L61

```cmd
cmd /C "set "NODE_OPTIONS=--require "c:/Program Files/Microsoft VS Code/resources/app/extensions/ms-vscode.js-debug/src/bootloader.js" --inspect-publish-uid=http" && set "VSCODE_INSPECTOR_OPTIONS={"inspectorIpc":"\\\\.\\pipe\\node-cdp.81396-aa5050a0-4.sock","deferredMode":false,"waitForDebugger":"","execPath":"C:\\Users\\user\\scoop\\apps\\nvm\\current\\nodejs\\nodejs\\node.exe","onlyEntrypoint":false,"autoAttachMode":"always","fileCallback":"C:\\Users\\user\\AppData\\Local\\Temp\\node-debug-callback-b17a11136f9e296d"}" && C:\Users\user\scoop\apps\nvm\current\nodejs\nodejs\node.exe node_modules/jest/bin/jest.js """c:/Users/user/git/examples/examples.test.ts""" -t """test with generated (.*?)""" --runInBand "
```

Maybe this arg should be based on `!isWindows()` instead?

**Review by domsleee — CHANGES_REQUESTED** [review]
please see comment, thanks

**Comment by domsleee** [comment]
Some more info I found, `?` is a wildcard character in `zsh`

```zsh
~ > echo ?
zsh: no matches found: ?
```

In bash, it just prints `?`.

So escaping provided by the vscode api does not escape the `?` (and nor should it I think?):
https://github.com/firsttris/vscode-jest-runner/blob/e3f7c76334c248ff426fd078a7634213b4106174/src/jestRunner.ts#L159

For powershell on windows, it is trying to match against literal double quotes, so it finds no matches (I don't think double quotes should be being used for powershell at all, but another problem).

Here is where `powershell` behaves differently to `bash`:
```powershell
❯ echo """abc"""
"abc"
```

In summary, it seems fine to use single quotes on linux to solve this issue
